src/domain/schemas/room-schemas.js

```diff
@@ -26,7 +27,8 @@ export const getRoomsQuerySchema = joi.object({
 export const postRoomBodySchema = joi.object({
   name: joi.string().required(),
   slug: slugSchema.required(),
-  isCollaborative: joi.boolean().required()
+  isCollaborative: joi.boolean().required(),
+  shortDescription: joi.string().allow('').required()
```

Comment:
It would be nice to have client side validation like on the name field in order to not get the server error (also there should be a "required" star in front of the label:

![Screenshot from 2023-03-28 18-31-09](https://user-images.githubusercontent.com/2676617/228308542-31c42200-b00e-4f95-96e0-99f6e7194f52.png)

As it is a required field, we might also want to push it up in the form to be below the "name" field?

Reply:
Hm, now I see, that empty string is allowed. I wonder why I get this message when trying to create a new room :thinking:

Reply:
Just like ChatGPT, I am at times hallucinating 🤦‍♀️ Somehow I lost a small change for creating rooms.
Thanks for catching it! Made the changes.

Reply:
:+1:

Reply:
Yep, I just saw, good idea! :+1: